mint.json

```diff
@@ -89,7 +89,8 @@
         "sql/scalar-functions",
         "sql/aggregate-functions",
         "sql/window-functions",
```

Comment:
did you mean to also include UDFs in this branch?

Reply:
No, branched off the wrong branch.